Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'database_cleaner', '~> 1.3.0', require: false
 group :development do
   gem 'guard'
   gem 'guard-rspec', require: false
+  gem 'pry-rescue'
```

Comment:
Do you want to leave this in?  I'm okay with it, just want to check that you intend it to be present.